pkg/R/RDD.R

```diff
@@ -23,7 +24,10 @@ setClass("PipelinedRDD",
 
 
 setMethod("initialize", "RDD", function(.Object, jrdd, serialized,
-                                        isCached, isCheckpointed) {
+                                        isCached, isCheckpointed, colNames) {
+  # Check that RDD constructor is using the correct version of serialized
+  stopifnot(class(serialized) == "character") 
```

Comment:
In addition to this check if will be good to check if `serialized` is equal to either `rows` or `bytes` or `string` just to handle cases where somebody might call it `row` instead of `rows` etc.